utils/spacewalk-hostname-rename

```diff
@@ -71,9 +71,51 @@ IPADDR_REGEX="($IPV4ADDR_REGEX)|($IPV6ADDR_REGEX)"
 
 ###############################################################################
 
+function default_or_input() {
+    local MSG="$1"
+    local VARIABLE="$2"
+    local DEFAULT="$3"
+    local SILENT="$4"
+
+    local INPUT
+    local CURRENT_VALUE=${!VARIABLE}
+    #in following code is used not so common expansion
+    #var_a=${var_b:-word}
+    #which is like: var_a = $var_b ? word
+    DEFAULT=${CURRENT_VALUE:-$DEFAULT}
+    local VARIABLE_ISSET=$(set | grep "^$VARIABLE=")
+
+    
+    echo -n "$MSG [$DEFAULT]: "
+
+    if [ -z "$VARIABLE_ISSET" ]; then
+        MANUAL_ANSWERS=1
+        read INPUT
```

Comment:
Probably not super relevant in this case, but it's always better to use `read -r` to [avoid interpreting backslashes](https://github.com/koalaman/shellcheck/wiki/SC2162).